### DISCUSSION TITLE
Reminder at frequent interval and add alarm as sound

### DISCUSSION THREAD

**Discussion by newpipemember** [iSoron/uhabits#1301] [Feature Requests]
I would request you to set an option to remind at a frequent interval like every 20-30 minutes to 1-2 hours. Also add an alarm option that will also serve in  alarm purpose.

**Comment by mohitxskull** [comment]
Yes i want this feature too

**Reply by Sudhegan** [reply]
yes this is what i am seaching so for but no apps for that,  and this feature is useful to create a new habit,
